.github/workflows/sql-workbench-release-workflow.yml

```diff
@@ -46,13 +46,13 @@ jobs:
         with:
           timeout_minutes: 60
           max_attempts: 3
-          command: cd kibana/plugins/sql-workbench; yarn kbn bootstrap
+          command: cd kibana/plugins/workbench; yarn kbn bootstrap
 
       - name: Build Artifact
         run: |
-          cd kibana/plugins/sql-workbench
+          cd kibana/plugins/workbench
           yarn build
           artifact=`ls ./build/*.zip`
 
-          aws s3 cp $artifact s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/kibana-plugins/opendistro-sql-workbench/
+          aws s3 cp $artifact s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/kibana-plugins/opendistro-query-workbench/
```

Comment:
`index.js` and `package.json` still use `sql-workbench`, where does `query-workbench` come from?

Reply:
refracted in index.js and package.json